apps/src/music/views/HeaderButtons.tsx

```diff
@@ -73,7 +79,10 @@ const HeaderButtons: React.FunctionComponent<HeaderButtonsProps> = ({
       <button
```

Comment:
Should we just not render the button at all in this case?

Reply:
Good call.  Will apply this to three of the button: refresh, undo, redo.